src/opera/pge/dswx_s1/dswx_s1_pge.py

```diff
@@ -27,6 +32,46 @@ class DSWxS1PreProcessorMixin(PreProcessorMixin):
 
     _pre_mixin_name = "DSWxS1PreProcessorMixin"
 
+    def _validate_algorithm_parameters_config(self):
+        """
+        The DSWx-S1 interface SAS uses two runconfig files; one for the main SAS,
+        and another for algorithm parameters.  This allows for independent modification
+        of algorithm parameters within it's own runconfig file. This method performs validation
+        of the 'algorithm parameters' runconfig file against its associated schema file. The SAS
+        section of the main runconfig defines the location within the container of the 'algorithm
+        parameters' runconfig file, under ['dynamic_ancillary_file_group']['algorithm_parameters'].
+        The schema file for the 'algorithm parameters' runconfig file is referenced under
+        ['PrimaryExecutable']['ParametersSchemaPath'] in the PGE section of the runconfig file.
+
+        """
+        self.runconfig = RunConfig(self.runconfig_path)
+
+        # Get the path to the 'algorithm_parameters_s1.schema.yaml' file
+        algorithm_parameters_schema_file_path = self.runconfig.algorithm_parameters_schema_path
+        if isfile(algorithm_parameters_schema_file_path):
+            # Load the 'algorithm parameters' schema
+            algorithm_parameters_schema = yamale.make_schema(algorithm_parameters_schema_file_path)
+        else:
+            raise RuntimeError(
+                f'Can not validate algorithm_parameters schema file.  '
+                f'File: ({algorithm_parameters_schema_file_path}) not found.'
+            )
+
+        # Get the 'algorithm parameters' runconfig file
+        self.algorithm_parameters_runconfig = self.runconfig.algorithm_parameters_config_path
+        if isfile(self.algorithm_parameters_runconfig):
+            # Load the 'algorithm parameters' runconfig file
+            algorithm_parameters_config_data = yamale.make_data(self.algorithm_parameters_runconfig)
+            pass
```

Comment:
This `pass` should be removed

Reply:
oops, hangover from when I was first setting things up

Reply:
Looks like this is still present after your latest changes